modules/ossm-config-openshift-monitoring.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/observability/ossm-observability-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+:context: ossm-config-openshift-monitoring
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. Content came from different places, so the first priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
```

Comment:
Is it possible to separate the `user-workload monitoring` configuration from the Kiali configuration? Users may want to configure `user-workload monitoring` but not want Kiali. This guide also doesn't use Kiali since the user simply queries user workload monitoring for `istio_requests_total`: https://github.com/openshift/openshift-docs/pull/80434/files#diff-37e6e97f3990f254fa9429eed5272f9251a744bd8c4e39b8061a2ed4d4b0ec67R145

It seems like this section could just be for configuring OSSM with `user-workload-monitoring` and you could have a separate section/guide for configuring `Kiali` with `user-workload-monitoring`.

Reply:
So configuring Kiali with user-workload monitoring ends at Step 2? And Step 3 and Step 4 are for configuring user-workload monitoring with just Service Mesh?

Reply:
I'm not sure on the exact step numbers anymore but looking at the upstream docs, [these steps](https://github.com/istio-ecosystem/sail-operator/tree/main/docs#scraping-metrics-using-the-openshift-monitoring-stack) are just for configuring Service Mesh with user-workload monitoring. And [these steps](https://github.com/istio-ecosystem/sail-operator/tree/main/docs#integrating-kiali-with-the-openshift-monitoring-stack) are for configuring Kiali. They are separate.

Reply:
Steps have been updated with the configuring Kiali content you linked to.